### DISCUSSION TITLE
File name for snaps based on userstring or clipped tiddler title

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#110] [General]
Can the file name for snaps be based on userstring or clipped tiddler title

**Comment by buggyj** [comment]
Calling the mediasaver addon from within a tiddlyclip rule, we have, eg,

```
#title=((*@pagetitle*))
$text=[img[((*@csaver(#title,@snap)*))]]
```

`#title` can be anything

**Reply by arunnbabu81** [reply]
@buggyj Thank you. Its working.

**Reply by arunnbabu81** [reply]
@buggyj Any suggetsions regarding the format used for tiddler titles. I was using `/` for titles . For example`Tweet/authorname/topicname`. But since filenames don't allow `/` , I had to change to `_`  (for example`Tweet_authorname_topicname`) for getting snaps with similar filenames for a particular tiddler. Is there any other symbols which are better ?

Which one do you prefer ?
`Tweet/authorname/topicname`
`Tweet.authorname.topicname`
`Tweet_authorname_topicname`

**Reply by buggyj** [reply]
xxx-yyy-zzz is the preferred format for file names, but underscores would work as well
